File: cases/taint/javascript/model-declared-sink-positive/Audit.js
function dfb_source() { // DFB-SOURCE: model-declared-sink-input
  return "tainted";
}

const Audit = {
  record: function record(value) {}, // DFB-SINK: model-declared-sink-sink
  discard: function discard(value) {}
};

function run() {
  Audit.record(dfb_source());
  Audit.discard("clean");
}

File: cases/taint/python/model-declared-sink-positive/audit.py
def dfb_source():  # DFB-SOURCE: model-declared-sink-input
    return "tainted"


def record(value):  # DFB-SINK: model-declared-sink-sink
    pass


def discard(value):
    pass


def run():
    record(dfb_source())
    discard("clean")
